opensearch/sycamore-opensearch.sh

```diff
@@ -570,15 +660,26 @@ setup_transient() {
     _curl "${BASE_URL}/_cluster/settings" \
     | grep -Fq aryn_deploy_complete && die "aryn_deploy_complete already set"
 
-    deploy_model "${EMBEDDING_MODEL_ID}" "${EMBEDDING_TASK_ID}" "embedding"
-    deploy_model "${OPENAI_MODEL_ID}" "${OPENAI_TASK_ID}" "OpenAI"
-    deploy_model "${RERANKING_MODEL_ID}" "${RERANKING_TASK_ID}" "reranking"
+    model_is_deployed "${EMBEDDING_MODEL_ID}" ||
+        deploy_model "${EMBEDDING_MODEL_ID}" "${EMBEDDING_TASK_ID}" "embedding"
```

Comment:
I think we should we modify deploy_model to start with
```
if model_is_deployed "$1"; then
  status "Model $1 for $3 already deployed"
  return 0
fi
```
Then these duplicated lines simplify down.